### DISCUSSION TITLE
Lesson 6;"Foundry Fund me" forge build error

### DISCUSSION THREAD

**Discussion by ghost** [Cyfrin/foundry-full-course-cu#252] [Q&A] [answered]
Hi there, 

I'm already add to my "**foundry.tml**" the remapping but I'm having the following error whe trying to **forge build**

"forge build
[⠢] Compiling...
No files changed, compilation skipped"

Please see attached:
<img width="674" alt="forge build" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/136945345/76b319a0-dc00-4a79-b76f-bc5829b84c4f">

**Comment by shikhar229169** [comment, marked as answer]
Hey @aesv22, actually you didn't saved your files. Consider pressing ```ctrl + s``` to save individual files.

**Reply by ghost** [reply]
Thank you very much @shikhar229169

**Comment by sharonphiliplima** [comment]
Hey there, I am facing a similar error:
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/69852010/d1d63aec-0e94-405d-95e5-e0f62128284c)
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/69852010/f1494004-3b14-491a-a859-cf77e54df062)

**Reply by devblixt** [reply]
1. Check if you have the AggregatorV3Interface contracts downloaded 
2. Check if the remapping path is correct. You may have a different folder layout so you will have to readjust the path

**Comment by cromewar** [comment]
@aesv22 please be sure you have saved the file on your editor, as you have the circle indicating you haven't saved the changes.
@sharonphiliplima please refer to #223 as @IAm0xTime suggested.

**Reply by ghost** [reply]
Thank you very much @cromewar
